src/wallet/wallet.h

```diff
@@ -82,6 +82,7 @@ class CTxMemPool;
 class CBlockPolicyEstimator;
 class CWalletTx;
 struct FeeCalculation;
+struct ReserveKey { int64_t index; bool internal; };
```

Comment:
could you rename this to be something that isn't so easily confused with `CReserveKey`